var/spack/repos/builtin/packages/py-morphology-repair-workflow/package.py

```diff
@@ -19,8 +19,20 @@ class PyMorphologyRepairWorkflow(PythonPackage):
 
     depends_on('py-setuptools', type=('build', 'run'))
 
-    depends_on('py-neuroc', type='run')
-    depends_on('py-morph-tool', type='run')
-    depends_on('py-neuror', type='run')
-    depends_on('py-more-itertools', type='run')
-    depends_on('py-morph-validator', type='run')
+    depends_on('py-click@7.0:', type='run')
+    depends_on('py-dask+bag@2.19.0:', type='run')
```

Comment:
Is this actually distributed or could this be done via multiprocessing?

Reply:
From what I can see `dask` in `morphology-repair-workflow` is used only in 
```
ok, failures = zip(*dask.bag.from_sequence(iterable).map(wrapped).compute())
```
There is no multi-node setup, but It should wrap multiprocessing.
Do you see any reason to avoid dask in this case? (In some other cases we used `joblib`)

Reply:
IMO, the question should more be why Dask and not just pure `multiprocessing`? Why add a complex dependency when `multiprocessing` could do it?

Same for `joblib`, if it's just to run stuff on a single node.

Reply:
+1 for multiprocessing if it is not distributed. I have a PR open on that code to fix dependencies with latest versions of other codes, I can remove dask dependency in the same time.